packages/documentation-framework/scripts/webpack/webpack.base.config.js

```diff
@@ -55,20 +55,12 @@ module.exports = (_env, argv) => {
             options: {
               cacheDirectory: '.cache/babel',
               cacheCompression: false,
-              presets: [['@babel/preset-env', {
-                loose: true,
-                corejs: 3,
-                useBuiltIns: 'entry',
-                exclude: ['transform-regenerator', 'transform-async-to-generator'],
-                modules: false,
-                targets: "> 0.25%, not dead"
-              }]],
-              plugins: [
-                '@babel/plugin-transform-react-jsx',
-                '@babel/plugin-proposal-class-properties',
-                '@babel/plugin-proposal-optional-chaining',
-                ["@babel/plugin-proposal-private-methods", { "loose": false }],
-                ["@babel/plugin-proposal-private-property-in-object", { "loose": false }]
+              presets: [
+                '@babel/preset-react',
+                ['@babel/preset-env', {
+                  loose: true,
+                  targets: '> 1%, not dead'
```

Comment:
I've narrowed the support target, as this [was matching](https://browsersl.ist/#q=%3E+0.25%25%2C+not+dead) a lot of unsupported browsers. The new target still has [wide support](https://browsersl.ist/#q=%3E+1%25%2C+not+dead) for various browsers, but prevents excessive transpilation.